README.md

```diff
@@ -30,6 +30,10 @@ This information reflects the head of this branch.
 | ------------------------------------------------------------------------------------------ | ----------------------------| --------------- | -------------------- |
 | [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/csi-snapshotter | 1.17         | v1beta1              |
 | [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/snapshot-controller | 1.17     | v1beta1              |
+| [CSI Spec v1.1.0](https://github.com/container-storage-interface/spec/releases/tag/v1.1.0) | quay.io/k8scsi/csi-snapshotter | 1.17         | v1beta1              |
+| [CSI Spec v1.1.0](https://github.com/container-storage-interface/spec/releases/tag/v1.1.0) | quay.io/k8scsi/snapshot-controller | 1.17     | v1beta1              |
+| [CSI Spec v1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | quay.io/k8scsi/csi-snapshotter | 1.17         | v1beta1              |
+| [CSI Spec v1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | quay.io/k8scsi/snapshot-controller | 1.17     | v1beta1              |
```

Comment:
Can you also update the image locations?
k8s.gcr.io/sig-storage/csi-snapshotter
k8s.gcr.io/sig-storage/snapshot-controller

Reply:
@xing-yang done.. thanks !